docs/self-managed/zeebe-deployment/security/client-authorization.md

```diff
@@ -170,6 +174,7 @@ Since there are several environment variables that can be used to configure an `
 - `ZEEBE_CLIENT_ID` - The client ID used to request an access token from the authorization server
 - `ZEEBE_CLIENT_SECRET` - The client secret used to request an access token from the authorization server
 - `ZEEBE_TOKEN_AUDIENCE` - The audience for which the token should be valid
+- `ZEEBE_TOKEN_SCOPE` - The [OAuth scope](https://oauth.net/2/scope/) which can be set optionally, not send if left unset
```

Comment:
I'm missing why someone would set this @megglos. Can you elaborate? I'm not sure if we need to add it to the docs.

Reply:
In case it's needed by the OAuth provider utilized, the trigger for introducing this param was Azure AD support getting added to identity, see https://github.com/camunda/zeebe/issues/15391#issue-2014821708